### ISSUE TITLE
All servers won't work well.

### ISSUE THREAD

**Issue by cryptcoin-junkey** [CounterpartyXCP/counterwallet#912]
AFAIK, all countewallet servers on the net don't work well.
It was overridden by curious download links.
This issue is reproduced not only on XCP version but also Monaparty's.
(And currently it seems Dogeparty's is safe.)

![Screenshot 2022-04-12 06 34 56](https://user-images.githubusercontent.com/16408160/162838031-7296e6c6-3f19-4548-8aff-3c311dc442d9.png)
![Screenshot 2022-04-12 06 34 05](https://user-images.githubusercontent.com/16408160/162838039-b1688191-6ca5-457c-a867-c2fbfe46e66f.png)

**Comment by wakiyamap** [comment]
It seems that `jReject` is doing something wrong.
```
../counterwallet/src/vendors/jReject/doc.html:  paragraph2: 'Just click on the icons to get to the download page',
../counterwallet/src/vendors/jReject/README.md:         paragraph2: 'Just click on the icons to get to the download page',
../counterwallet/src/vendors/jReject/js/jquery.reject.js:               paragraph2: 'Just click on the icons to get to the download page',
../counterwallet/src/vendors/jReject-ie11/index.js:             paragraph2: 'Just click on the icons to get to the download page',
```

**Comment by jdogresorg** [comment]
fixed by #910 and #913
